packages/template-block/src/TemplateBlock.spec.ct.tsx

```diff
@@ -9,15 +9,20 @@ import {
     PreviewDisplayType,
```

Comment:
Not for this PR, but I think it would be nice to have some small keyboard accessibility tests, if everything within the block is reachable by keyboard (e.g. changing title, description, button etc)